package.json

```diff
@@ -26,13 +26,14 @@
     "generate": "stencil generate"
   },
   "dependencies": {
-    "@stencil/core": "^2.13.0"
+    "@stencil/core": "^3.0.0",
+    "@types/node": "^16.18.11"
```

Comment:
Can these just be dev deps? That's what I had them as when we talked about this earlier

Reply:
D'Oh - that's my bad. I guess I forgot to save `@types/node` with `-D` 😢 Updated.

I don't believe we can move `@stencil/core`, as some output targets require it as a production dependency.